pegasus/sites.v3/csedweek.org/views/footer.haml

```diff
@@ -1,7 +1,7 @@
 %div{:style=>'width: 100%; height: 80px; padding: 20px; overflow: hidden; background-color: rgb(4, 126, 188)'}
   %div{:class=>'whitefooter', :style=>'width: 970px; margin: 0 auto;'}
     %div{:style=>'float:left'}
-      %a{:href=>"https://code.org/privacy", :class=>"whitefooterlink", :target=>"_blank"} Privacy Policy:
+      %a{:href=>"https://code.org/privacy", :class=>"whitefooterlink", :target=>"_blank"} Privacy Policy
```

Comment:
Perhaps we could include `rel="noopener noreferrer"` too, per [this kind of discussion](https://github.com/code-dot-org/code-dot-org/pull/36943#discussion_r496260009).